packages/platform/utils/permissions.ts

```diff
@@ -9,3 +18,36 @@ export const hasPermissions = (userPermissions: number, permissions: PLATFORM_PE
   // use bitwise AND to check if each required permission is present
   return permissions.every((permission) => hasPermission(userPermissions, permission));
 };
+
+export const hasEventTypeReadPermission = (userPermissions: number): boolean => {
```

Comment:
Shall we have `utils/permissions` as a folder and then sub-folders for booking, event-type and schedule e.g. `utils/permissions/booking` ?

It's okaish now to keep track of 3 permission groups but if we would add more stuff than booking, event-type and schedule then its hard to navigate everything in 1 file.

What do you think?

Reply:
Then we would also have smaller test file as each test group would be in its own folder

Reply:
let's see how it goes, we can do it when the need appears